docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source.md

```diff
@@ -0,0 +1,124 @@
+---
+id: zero-networks-segment-source
+title: Zero Networks Segment Source
+sidebar_label: Zero Networks Segment
+description: Learn how to collect audit logs and network activity data from Zero Networks Segment.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/zero-networks-icon.png')} alt="zero-networks-icon" width="40" />
+
+Zero Networks is a cybersecurity company that provides cloud-based security solutions for your networks and endpoints. Segment is a solution that aims to provide comprehensive network security by implementing zero-trust principles. With Segment, Zero Networks provides a cloud-based platform that allows you to create micro-segments across the network. These micro-segments are small, isolated portions of the network that are tightly controlled and can only be accessed by authorized users and devices.
+
+This source ingests audit logs and network activities via Zero Networks Segment API.
+
+## Data Source
+
+- **Audit Logs**. A set of audit events for everything that happens in the product.
+- **Network Activities**. A network connection that occurs in the customer environment which includes properties from both source and destination.
+
+## Setup and Configuration
+
+The Zero Networks Segment source requires you to provide the API key and Network Activity Filters (optional) to access the data.
+
+- Sign in to the Zero Networks platform and create an API Key.
+- (Optional) Follow the below steps to create Network Activity Filters:
```

Comment:
```suggestion
1. Sign in to the Zero Networks platform and create an API Key.
1. (Optional) Follow the below steps to create Network Activity Filters:
```

Reply:
@kimsauce Both points are independent of each other and not a continuation. So step numbers are not added.